### ISSUE TITLE
AttributeError after meeting import

### ISSUE THREAD

**Issue by emanuelschuetze** [OpenSlides/openslides-backend#1030]
Try to import an existing, exported OS3 meeting into OS4:

```
Error handling request /system/action/handle_request
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/gunicorn/workers/sync.py", line 136, in handle
    self.handle_request(listener, req, client, addr)
  File "/usr/local/lib/python3.8/site-packages/gunicorn/workers/sync.py", line 179, in handle_request
    respiter = self.wsgi(environ, resp.start_response)
  File "/app/openslides_backend/http/application.py", line 82, in __call__
    return self.wsgi_application(environ, start_response)
  File "/app/openslides_backend/http/application.py", line 72, in wsgi_application
    response = self.dispatch_request(request)
  File "/app/openslides_backend/http/application.py", line 38, in dispatch_request
    response_body, access_token = view_instance.dispatch(request)
  File "/app/openslides_backend/http/views/base_view.py", line 101, in dispatch
    return func(request)
  File "/app/openslides_backend/http/views/action_view.py", line 39, in action_route
    response = handler.handle_request(request.json, user_id, is_atomic)
  File "/app/openslides_backend/action/action_handler.py", line 106, in handle_request
    results = self.execute_write_requests(self.parse_actions, payload)
  File "/app/openslides_backend/action/action_handler.py", line 143, in execute_write_requests
    write_requests, data = get_write_requests(*args)
  File "/app/openslides_backend/action/action_handler.py", line 180, in parse_actions
    write_request, results = self.perform_action(element, relation_manager)
  File "/app/openslides_backend/action/action_handler.py", line 215, in perform_action
    write_request, results = action.perform(action_data, self.user_id)
  File "/app/openslides_backend/action/actions/meeting/import_.py", line 65, in perform
    instance = self.base_update_instance(instance)
  File "/app/openslides_backend/action/action.py", line 282, in base_update_instance
    return self.update_instance(instance)
  File "/app/openslides_backend/action/actions/meeting/import_.py", line 75, in update_instance
    if not len(meeting_json.get("meeting", {}).values()) == 1:
AttributeError: 'list' object has no attribute 'values'
```

**Comment by reiterl** [comment]
It seems like the OS3 meeting export uses a different format like the OS4 meeting.import.

**Comment by jsangmeister** [comment]
This should be resolved now after #1055 was merged. @emanuelschuetze can you verify this or report if more problems occur?

**Comment by emanuelschuetze** [comment]
works now with current master
